packages/blocks/src/page-block/edgeless/components/edgeless-selected-rect.ts

```diff
@@ -58,6 +58,9 @@ export class EdgelessSelectedRect extends WithDisposable(LitElement) {
   @property({ type: SurfaceManager })
   surface!: SurfaceManager;
 
+  @property()
+  shift!: boolean;
```

Comment:
The naming of this variable is confusing, you need to use clearer naming or add comments to it.